### PR TITLE
Upgrade spc module second version

### PR DESCRIPTION
**Pull request by OSsunNO**
AC:

- Implement the SPC module according to the option 4
   - option 4
   -> replace emoji with 'explanation text', replace emoticon with 'category', replace special - character with '')
   -> print out detailed type for each detected spc -> i.e. desc='emoji' / i.e. desc='emoticon / i.e. desc='spc' 
- Define the emoticon_dict_v1 for emoticon

### CODE REVIEW

**Comment by OSsunNO** [comment]
- emoticon_dict_v1 on google drive
   -> [emoticon_dict_v1](https://drive.google.com/file/d/1oPHImP8ByaZ7r9ES9ECzUfcaEcifw3Hq/view?usp=sharing)

**Comment by OSsunNO** [comment]
image
image

File: dt-demo-be/modules/spc/run.py [hunk 1]
```diff
@@ -35,10 +36,34 @@ def run(fileName):
...
+                # EMOTICON
+                # TODO: emoticon(spc combination) processing
+                for e, c in zip(emoticon_dict['emoticon'], emoticon_dict['category']):
+                    if e in target:
+                        print(f"detected_emoticon: {e}, converted_text_form: {c}, desc: emoticon")  # print out the log
+                        target = target.replace(e, ':'+c+':')  # replace the emoticon with :category:
+                
+                
+                # 1-digit SPC
+                # TODO: 1-digit spc processing
```
Comment (doorooful) [inline dt-demo-be/modules/spc/run.py:57]: Can you remove this line?

File: dt-demo-be/modules/spc/run.py [hunk 2]
```diff
@@ -35,10 +36,34 @@ def run(fileName):
...
+                for i in range(len(a)):
+                    print(f"detected_emoji: {a[i].chars}, converted_text_form: {emoji.demojize(a[i].chars)}, desc: emoji") # print out the log
+                # let emoji convert into text form
                 target=emoji.demojize(target)
-                for s in spc_dict['spc']:
-                    target = target.replace(s, '')
+                
+                # EMOTICON
+                # TODO: emoticon(spc combination) processing
+                for e, c in zip(emoticon_dict['emoticon'], emoticon_dict['category']):
```
Comment (doorooful) [inline dt-demo-be/modules/spc/run.py:50]: Same here!

**Review by doorooful — COMMENTED** [review]
Please rename dict excel file!
Thank you

**Comment by OSsunNO** [comment]
> Please rename dict excel file! Thank you

I resolved it, thank you so much for your feedback :)

**Review by doorooful — APPROVED** [review]
Please attach dict excel file on the comment 😀 Thanks!!

**Comment by OSsunNO** [comment]
> Please attach dict excel file on the comment 😀 Thanks!!

I uploaded the link for excel file in the first comment! 😃 Thank you!

**Pull request merged**